### PR TITLE
Readme - split out getting help / contributing back & add dev docs

### PR DESCRIPTION
**Pull request by lb-**
- Add basic development docs so it is clearer that this project uses `yarn` and what basic scripts are needed to get started locally.
- Note: I still cannot get the local `yarn start` server with the examples working but have assumed I am doing something wrong so have included that instruction also.

### CODE REVIEW

**Comment by marcoroth** [comment]
Hey @lb-, thanks for opening this pull request!

This looks good to me, though I'm still curious why you are not able the get `yarn start` to work. Are you getting any errors?

File: README.md [hunk 1]
```diff
@@ -44,16 +44,27 @@ You can use Stimulus with any asset packaging systems. And if you prefer no buil
...
 We expect all Stimulus contributors to abide by the terms of our [Code of Conduct](CODE_OF_CONDUCT.md).
 
+### Development
+
+- Fork the project locally
+- `yarn install`
+- `yarn start` - to run the local dev server with examples
+- `yarn test` - to run the unit tests
+- `yarn lint` - to run the linter with Eslint
+- `yarn format` - to format changes with Pettier
```
Comment (marcoroth) [inline README.md:66]:
```suggestion
- `yarn lint` - to run the linter with ESLint
- `yarn format` - to format changes with Prettier
```

Comment (lb-) [reply README.md]: Thanks.

**Comment by lb-** [comment]
> ... I'm still curious why you are not able the get `yarn start` to work. Are you getting any errors?

I'll raise a new issue maybe to help explain the problem.

**Review by marcoroth — APPROVED** [review]

**Pull request merged**